yb-voyager/src/dbzm/valueConverter.go

```diff
@@ -202,20 +208,20 @@ func (conv *DebeziumValueConverter) convertMap(eventSchema string, tableName str
 			continue
 		}
 		columnValue := *value
-		colType, err := schemaRegistry.GetColumnType(tableNameInSchemaRegistry, column, conv.shouldFormatAsPerSourceDatatypes())
+		colType, colDbzmSchema, err := schemaRegistry.GetColumnType(tableNameInSchemaRegistry, column, conv.shouldFormatAsPerSourceDatatypes())
 		if err != nil {
 			return fmt.Errorf("fetch column schema: %w", err)
 		}
 		if !checkSourceExporter(exportSourceType) && strings.EqualFold(colType, "io.debezium.time.Interval") {
-			colType, err = conv.schemaRegistrySource.GetColumnType(strings.ToUpper(tableName), strings.ToUpper(column), conv.shouldFormatAsPerSourceDatatypes())
+			colType, colDbzmSchema, err = conv.schemaRegistrySource.GetColumnType(strings.ToUpper(tableName), strings.ToUpper(column), conv.shouldFormatAsPerSourceDatatypes())
 			//assuming table name/column name is case insensitive TODO: handle this case sensitivity properly
 			if err != nil {
 				return fmt.Errorf("fetch column schema: %w", err)
 			}
 		}
 		converterFn := conv.valueConverterSuite[colType]
 		if converterFn != nil {
-			columnValue, err = converterFn(columnValue, formatIfRequired)
+			columnValue, err = converterFn(columnValue, formatIfRequired, colDbzmSchema) // Fix: Pass the value of colDbzmSchema instead of the pointer
```

Comment:
Why the `Fix:` comment? We should be passing pointer.

Reply:
Yes, nothing needs to be done here, not sure how this comment is here probably github copilot. Will remove it.